app/models/manageiq/providers/redhat/infra_manager/supported_apis_mixin.rb

```diff
@@ -0,0 +1,85 @@
+module ManageIQ::Providers::Redhat::InfraManager::SupportedApisMixin
+  def supported_api_versions
+    return supported_api_versions_from_cache if api_version.blank?
```

Comment:
The cache is probably no longer convenient. It was convenient when determining the supported versions was expensive because it required talking to the server. But now that it only needs access to the database I think we can just remove completely the caching code.

Reply:
done